packages/lib/src/test/helpers/assertRevert.ts

```diff
@@ -1,10 +1,12 @@
-export default async function assertRevert(promise, invariants = () => {}) {
+import assert from 'assert';
+
+export default async function assertRevert(promise:Promise<any>, invariants = ():void => {return;}) {
```

Comment:
Can we avoid the `return` statement in these cases?

Reply:
I'm pretty sure we can :laughing: 
That definitely looks weird... let me see what I can do.

Reply:
Done.